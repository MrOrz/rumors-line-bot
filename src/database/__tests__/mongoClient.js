import Client from '../mongoClient';

describe('MongoClient', () => {
  it('should connect', async () => {
    const client = await Client.getInstance();
    expect(client.mongoClient.isConnected()).toMatchSnapshot();
    expect(client.db.databaseName).toMatchSnapshot();
  });
});
