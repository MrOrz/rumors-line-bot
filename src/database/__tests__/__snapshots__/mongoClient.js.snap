// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MongoClient should connect 1`] = `true`;

exports[`MongoClient should connect 2`] = `"cofacts"`;
